scim-server-examples/scim-server-spring-boot/README.md

```diff
@@ -0,0 +1,20 @@
+Apache Directory SCIMple In Memory Example
+==========================================
+
+This example project demo's how to:
+
+* Add a custom SCIM Extension
+* Manage Users and Groups (in memory)
+
+Use this as a starter point on how to integrate Apache Directory SCIMple into your own project.
+
+Run: `mvn spring-boot:run` and then access one of the endpoints:
+
+```bash
+# httpie
+http :8080/Users
```

Comment:
Should this be capitalized?

Reply:
Oddly enough, yes: https://www.rfc-editor.org/rfc/rfc7644#section-3.2

For some reason that capitalization seems really odd to me, but it's in the spec 🤷